src/sagemaker/workflow/pipeline.py

```diff
@@ -105,7 +110,9 @@ def to_request(self) -> RequestType:
             "PipelineExperimentConfig": self.pipeline_experiment_config.to_request()
             if self.pipeline_experiment_config is not None
             else None,
-            "Steps": build_steps(self.steps, self.name),
+            "Steps": build_steps(
+                self.steps, self.name, self.pipeline_definition_config.use_custom_job_prefix
```

Comment:
why not pass the entire pipeline_definition_config struct here?

Reply:
Good call — let’s do that to clean it up. Do we think a feature flag via the config is the best way to do this ? I had theorized with the more concrete **kwargs but it puts more responsibility on the customer to build the params correctly as we expand this config into the future.